packages/manager/modules/nasha/src/partition/access/add/styles.less

```diff
@@ -1,3 +1,7 @@
+@cloud_info_bg_color: #dfeaf8;
```

Comment:
Theses variables couldn't be scoped inside the `.nasha-partition-access-add` selector?